components/atom/button/src/index.js

```diff
@@ -0,0 +1,98 @@
+import React, {PropTypes} from 'react'
+import cx from 'classnames'
+
+const CLASS = 'sui-AtomButton'
+const TYPES = ['primary', 'accent', 'secondary', 'tertiary']
+const MODIFIERS = ['disabled', 'small', 'large', 'fullWidth', 'focused']
```

Comment:
I don't think all they are the same. My point is: `disabled` and `focused` are states, and the other ones are modifiers (related to the button size).

Reply:
It's a matter of naming. TYPES are exclusive. So if none if none is set, primary is added by default.
MODIFIERS are just agrregable classes. Event if disable is a state for native button, in this context it's simply a CSS modifier to change the style. The component has no state.